visidata/vd.py

```diff
@@ -449,7 +474,7 @@ def columnsMatch(sheet, row, columns, func):
             if columnsMatch(sheet, sheet.rows[r], columns, self.lastRegex.search):
                 if moveCursor:
                     sheet.cursorRowIndex = r
-                    status('search wrapped')   # the only reason for the duplicate code block
+                    status('search wrapped')   # the only reason for the duplicate code block TODO: this can be condensed.
```

Comment:
Agreed, as per our discussion earlier today. Will go through code again and deal with TODOs after docstrings are merged.